docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -0,0 +1,372 @@
+
+<!--Should library panel still be here since it's not in a Share modal anymore and we have a whole page for this? -->
+
+{{< admonition type="note" >}}
+If you change a panel, ensure that you save the changes before sharing.
+{{< /admonition >}}
+
+### Share an internal link
+
+To share a personalized, direct link to your panel within your organization, follow these steps:
+
+1. Hover over any part of the panel you want to share to display the actions menu on the top right corner.
+1. Click the menu and select **Share link**.
+1. (Optional) In the **Link settings** drawer that opens, set the following options:
+   - **Lock time range** - Change the current relative time range to an absolute time range. This option is enabled by default.
+   - **Shorten link** - Shorten the panel link. This option is disabled by default.
+1. Select the theme for the dashboard. Choose from **Current**, **Dark**, or **Light**.
+1. Do one or both of the following:
+   - Click **Copy link**.
+   - Click **Render image**, which [renders the panel as a PNG image](ref:image-rendering).
+1. Send the copied link or image URL to a Grafana user with authorization to view it.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+#### Query string parameters for server-side rendered images
+
+When you click **Render image** in the panel link settings, Grafana generates a PNG image of the panel with the following default parameters:
+
+| Parameter | Description                                                                                                                                                |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| width     | Width in pixels. Default is 800.                                                                                                                           |
+| height    | Height in pixels. Default is 400.                                                                                                                          |
+| tz        | Timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC.                                                          |
+| timeout   | Number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.                                         |
+| scale     | Numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+. |
+
+You can also update these parameters in the [image rendering configuration](ref:image-rendering-config).
+
+The following example shows a link to a server-side rendered PNG:
+
+```bash
+https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&from=1568719680173&to=1568726880174&panelId=4&fullscreen
+```
+
```

Comment:
The Grafana Play link above wasn't valid anymore so I put in a new one, but when I generate a new one, the URL is suuuuper long and the Shorten link option doesn't help. Is there a way to make these links shorter?

Reply:
Why the shorter link doesn't help? I've generated [this one](https://play.grafana.org/goto/LDBW8z6SR?orgId=1). Also, is it not possible to hide the link in the markup?

Reply:
It looks like the idea was to actually show what the HTML link looks like rather than embedding it in text.

Reply:
The link you provided seems to just be a dashboard link though. This is meant to be a link for a panel rendered as an image.

Reply:
You are right; no, we don't support shortened links for rendering panels.